### PR TITLE
[Components with more than 1 service] LPS-185474 Refactor components from layout module

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://issues.liferay.com/browse/LPS-185474

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:echo

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/template/template-service/src/main/java/com/liferay/template/internal/info/item/provider/TemplateInfoItemCapabilitiesProviderTracker.java [hunk 1]
```diff
@@ -91,12 +91,14 @@ protected void deactivate() {
...
-	private TemplateInfoItemCapability _templateInfoItemCapability;
-
 	@Reference
 	private TemplateNodeFactory _templateNodeFactory;
 
+	@Reference(
+		target = "(info.item.capability.key=" + TemplateInfoItemCapability.KEY + ")"
+	)
+	private InfoItemCapability _templatePageInfoItemCapability;
+
```
Comment (rafaprax) [inline modules/apps/template/template-service/src/main/java/com/liferay/template/internal/info/item/provider/TemplateInfoItemCapabilitiesProviderTracker.java:101]: Hey @MrJohn1911, please rename it to `_templateInfoItemCapability `. Thx

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemCapabilitiesProvider.java [hunk 2]
```diff
@@ -47,6 +46,6 @@ public List<InfoItemCapability> getInfoItemCapabilities() {
 
 	private final InfoItemCapability _displayPageInfoItemCapability;
 	private final InfoItemCapability _editPageInfoItemCapability;
-	private final TemplateInfoItemCapability _templatePageInfoItemCapability;
+	private final InfoItemCapability _templatePageInfoItemCapability;
```
Comment (rafaprax) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemCapabilitiesProvider.java:49]: Hey @MrJohn1911, please rename it to `_templateInfoItemCapability `. Thx

Comment (rafaprax) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemCapabilitiesProvider.java:49]: Check other usages as well. :)

File: modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/AddInfoItemStrutsActionValidationTest.java [hunk 3]
```diff
@@ -609,8 +610,10 @@ private void _markInputFragmentEntryLinkAsRequired(Layout layout)
 	@Inject
 	private CompanyLocalService _companyLocalService;
 
-	@Inject
-	private EditPageInfoItemCapability _editPageInfoItemCapability;
+	@Inject(
+		filter = "info.item.capability.key=" + EditPageInfoItemCapability.KEY
+	)
+	private InfoItemCapability _editPageInfoItemCapability;
```
Comment (rafaprax) [inline modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/AddInfoItemStrutsActionValidationTest.java:616]: Hey @MrJohn1911, please rename it to `_editInfoItemCapability`. Thx

Comment (rafaprax) [reply modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/AddInfoItemStrutsActionValidationTest.java:616]: Check other usages as well. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/AddInfoItemStrutsActionValidationTest.java (hunk 3 above)
Comment (rafaprax) [reply modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/AddInfoItemStrutsActionValidationTest.java:616]: never mind

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
This last change just modified the name of the field `_templatePageInfoItemCapability` to `_templateInfoItemCapability`.

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileImpl.java [hunk 4]
```diff
@@ -862,9 +862,8 @@ protected void verifyNotBeforeDateTime(
 
 		throw new AssertionException(
 			StringBundler.concat(
-				"Date ", nowDateTime.toString(), " is before ",
-				lowerBoundDateTime.toString(), " including clock skew ",
-				clockSkew));
+				"Date ", nowDateTime, " is before ", lowerBoundDateTime,
```
Comment (rafaprax) [inline modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileImpl.java:865]: Hey @MrJohn1911, it is not related to your changes. Please remove these changes from your pr. Thx

File: modules/test/portlet-title-localization-test/src/testIntegration/java/com/liferay/portlet/title/localization/test/PortletTitleTest.java [hunk 5]
```diff
@@ -74,7 +74,7 @@ public void testPortletTitles() throws Exception {
 		Assert.assertTrue(
 			StringBundler.concat(
 				"Please update the Language.properties files for the ",
-				"following portlets: ", portletIdsWithMissingTitles.toString()),
+				"following portlets: ", portletIdsWithMissingTitles),
```
Comment (rafaprax) [inline modules/test/portlet-title-localization-test/src/testIntegration/java/com/liferay/portlet/title/localization/test/PortletTitleTest.java:77]: Hey @MrJohn1911, it is not related to your changes. Please remove these changes from your pr. Thx

**Review by rafaprax — APPROVED** [review]

File: modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileImpl.java (hunk 4 above)
Comment (MrJohn1911) [reply modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileImpl.java]: @rafaprax Done!

File: modules/test/portlet-title-localization-test/src/testIntegration/java/com/liferay/portlet/title/localization/test/PortletTitleTest.java (hunk 5 above)
Comment (MrJohn1911) [reply modules/test/portlet-title-localization-test/src/testIntegration/java/com/liferay/portlet/title/localization/test/PortletTitleTest.java]: @rafaprax Done!

**Comment by MrJohn1911** [comment]
In this last push, I removed two classes that were not related to my pr.

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:forward:force

cc @shuyangzhou @tinatian

**Comment by dantewang** [comment]
Manually sent to https://github.com/brianchandotcom/liferay-portal/pull/135865

**Pull request closed without merging**